### DISCUSSION TITLE
setup with proscan

### DISCUSSION THREAD

**Discussion by jimgb17** [chuot/rdio-scanner#380] [Q&A]
hi all i am trying to setup rdio-scanner with proscan
is their a tutorial i can use i am new to all this i have sds200 scanner working with the latest full version of proscan i have both analog and digital frqs 

thanks in advance..

**Comment by rdmcvicar** [comment]
I was able to get going with this information: https://github.com/chuot/rdio-scanner/discussions/192#discussioncomment-2829014 -- basically the ProScan custom filename format and rdio-scanner dirwatch masks.

A couple side notes:

- If anyone knows a way (or feels like writing an enhancement request) to add something like #TGNAME to extract talkgroup name in addition to the #TGLBL that's already available, that'd be useful for bringing in ProScan traffic.

- I mainly use trunk-recorder and RTLSDR-airband for ingest, but have played a bit with ProScan to bring yet more traffic into https://scan-wis.net (so anyone in Wisconsin or nearby wanting to collaborate, please feel free to write to the email support link on the player).

Loving rdio-scanner,

Rich M.
